### DISCUSSION TITLE
Steins;Gate Theme (Landscape) [3.5"]

### DISCUSSION THREAD

**Discussion by AlexTheLugi** [mathoudebine/turing-smart-screen-python#376] [Themes]
Made to fit next to a monitor setup with RED_SIM's wallpapers from wallpaper engine: https://steamcommunity.com/sharedfiles/filedetails/?id=1524074929 

Pretty much based on @mathoudebine 's LandscapeEarth, changed the wallpaper and some information displayed. 
Text might be a bit hard to read because the color is maybe too similar to the background, but I couldn't find a way to make it better and didn't want to spend too much time on this.
Hope you like it!

![preview](https://github.com/mathoudebine/turing-smart-screen-python/assets/149326204/c7f9341b-0e0c-47e2-abb9-ba1b915f8df1)

Download: 
[SteinsGateTheme.zip](https://github.com/mathoudebine/turing-smart-screen-python/files/13198867/SteinsGateTheme.zip)

**Comment by mathoudebine** [comment]
Nice work, thanks for sharing!
